jenkins/pipelines/cd/tiup/tiup-mirror-update-multi-products.groovy

```diff
@@ -83,28 +83,28 @@ def pack = { name, version, os, arch ->
     if (name == "tidb") {
         sh """
         tiup package ${name}-server -C bin --name=${name} --release=${version} --entry=${name}-server --os=${os} --arch=${arch} --desc="${tidb_desc}"
-        tiup mirror publish ${name} ${tidb_version} package/${name}-${version}-${os}-${arch}.tar.gz ${name}-server --arch ${arch} --os ${os} --desc="${tidb_desc}"
+        tiup tiup:nightly mirror publish ${name} ${tidb_version} package/${name}-${version}-${os}-${arch}.tar.gz ${name}-server --arch ${arch} --os ${os} --desc="${tidb_desc}"
```

Comment:
Remember to create another PR to use product version.

Reply:
added a issue to trace it